src/Symfony/Component/Cache/Adapter/PdoAdapter.php

```diff
@@ -56,9 +56,9 @@ class PdoAdapter extends AbstractAdapter
      * @param int                    $defaultLifetime
      * @param array                  $options         An associative array of options
      *
-     * @throws InvalidArgumentException When first argument is not PDO nor Connection nor string
-     * @throws InvalidArgumentException When PDO error mode is not PDO::ERRMODE_EXCEPTION
-     * @throws InvalidArgumentException When namespace contains invalid characters
+     * @throws \InvalidArgumentException When first argument is not PDO nor Connection nor string
+     * @throws \InvalidArgumentException When PDO error mode is not PDO::ERRMODE_EXCEPTION
+     * @throws \InvalidArgumentException When namespace contains invalid characters
```

Comment:
Ah you are right! Did not see the `use Symfony\Component\Cache\Exception\InvalidArgumentException` :yum: